build.mjs fails on ARM64 with unsupported platform error

On a Surface Laptop 7:

```
PS C:\Users\danch\source\repos\MudBlazor\src> dotnet build
Restore complete (1.0s)
  MudBlazor.Analyzers netstandard2.0 succeeded (0.1s) → MudBlazor.Analyzers\bin\Debug\netstandard2.0\MudBlazor.Analyzers.dll
  MudBlazor.Examples.Data net10.0 succeeded (0.1s) → MudBlazor.Examples.Data\bin\Debug\net10.0\MudBlazor.Examples.Data.dll
  MudBlazor.Analyzers.CodeFixes netstandard2.0 succeeded (0.1s) → MudBlazor.Analyzers.CodeFixes\bin\Debug\netstandard2.0\MudBlazor.Analyzers.CodeFixes.dll
  MudBlazor failed with 2 error(s) (0.8s)
    EXEC : error Unsupported platform
    C:\Users\danch\source\repos\MudBlazor\src\MudBlazor\MudBlazor.csproj(122,5): error MSB3073: The command "dotnet tool exec BunDotNet.Cli@1.2.0 --yes -- wrapper --version 1.3.6 --path ../../tools --silent -- run build.mjs" exited with code -1.
  MudBlazor net10.0 failed with 2 error(s) (0.7s)
    EXEC : error Unsupported platform
    C:\Users\danch\source\repos\MudBlazor\src\MudBlazor\MudBlazor.csproj(138,5): error MSB3073: The command "dotnet tool exec BunDotNet.Cli@1.2.0 --yes -- wrapper --version 1.3.6 --path ../../tools --silent -- run build.mjs" exited with code -1.
                                                                                                           
Build failed with 4 error(s) in 2.8s                                                                       
```

--- a/.config/dotnet-tools.json
+++ b/.config/dotnet-tools.json
@@ -3,7 +3,7 @@
   "isRoot": true,
   "tools": {
     "bundotnet.cli": {
-      "version": "1.2.0",
+      "version": "1.3.0",
       "commands": [
         "bun"
       ],

--- a/src/Directory.Build.props
+++ b/src/Directory.Build.props
@@ -13,7 +13,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <BunVersion>1.3.9</BunVersion>
+    <BunVersion>1.3.10</BunVersion>
     <BunToolRestore>dotnet tool restore --tool-manifest ../../.config/dotnet-tools.json</BunToolRestore>
     <BunWrapper>dotnet tool run bun -- wrapper --version $(BunVersion) --path ../../tools --silent --</BunWrapper>
   </PropertyGroup>

